cfgov/unprocessed/js/modules/sample.js

```diff
@@ -0,0 +1,10 @@
+/* ==========================================================================
+   sample: sample JS for setting up tests. This is not included in
```

Comment:
TRIVIAL: could follow grammatical rules in regard to capitalization. `sample.js: Sample JS for setting up tests. This is not included in`.